presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PrintOutput.java

```diff
@@ -0,0 +1,316 @@
+                output.append('\n');
+            }
+        }
+    }
+
+    private static String indentMultilineString(String string, int level)
+    {
+        return string.replaceAll("(?m)^", indentString(level));
+    }
+
+    public String getTextOutput()
+    {
+        writeCurrentNodes();
+        return output.toString();
+    }
+
+    private void printPlanNodesStatsAndCost(StringBuilder output, List<PlanNode> nodes)
+    {
+        if (nodes.stream().allMatch(this::isPlanNodeStatsAndCostsUnknown)) {
+            return;
+        }
+
+        output.append(format("Cost: %s\n", nodes.stream().map(this::formatPlanNodeStatsAndCost).collect(joining("/"))));
+    }
+
+    private boolean isPlanNodeStatsAndCostsUnknown(PlanNode node)
+    {
+        PlanNodeStatsEstimate stats = estimatedStatsAndCosts.getStats().getOrDefault(node.getId(), PlanNodeStatsEstimate.unknown());
+        PlanNodeCostEstimate cost = estimatedStatsAndCosts.getCosts().getOrDefault(node.getId(), PlanNodeCostEstimate.unknown());
+        return stats.isOutputRowCountUnknown() || cost.equals(PlanNodeCostEstimate.unknown());
+    }
+
+    private String formatPlanNodeStatsAndCost(PlanNode node)
+    {
+        PlanNodeStatsEstimate stats = estimatedStatsAndCosts.getStats().getOrDefault(node.getId(), PlanNodeStatsEstimate.unknown());
+        PlanNodeCostEstimate cost = estimatedStatsAndCosts.getCosts().getOrDefault(node.getId(), PlanNodeCostEstimate.unknown());
+        return format("{rows: %s (%s), cpu: %s, memory: %s, network: %s}",
+                formatAsLong(stats.getOutputRowCount()),
+                formatEstimateAsDataSize(stats.getOutputSizeInBytes(node.getOutputSymbols(), types)),
+                formatDouble(cost.getCpuCost()),
+                formatDouble(cost.getMemoryCost()),
+                formatDouble(cost.getNetworkCost()));
+    }
+
+    private void printStats(StringBuilder output, List<PlanNode> nodes)
+    {
+        if (!stats.isPresent()) {
+            return;
+        }
+
+        PlanNode leafNode = nodes.get(0);
+        PlanNode rootNode = nodes.get(nodes.size() - 1);
+
+        long totalScheduledMillis = stats.get().values().stream()
+                .mapToLong(planNode -> planNode.getPlanNodeScheduledTime().toMillis())
+                .sum();
+
+        long totalCpuMillis = stats.get().values().stream()
+                .mapToLong(planNode -> planNode.getPlanNodeCpuTime().toMillis())
+                .sum();
+
+        PlanNodeStats nodeStats = stats.get().get(rootNode.getId());
+        if (nodeStats == null) {
+            output.append("Cost: ?, Output: ? rows (?B)\n");
+            return;
+        }
+
+        double scheduledTimeFraction = 100.0d * nodeStats.getPlanNodeScheduledTime().toMillis() / totalScheduledMillis;
+        double cpuTimeFraction = 100.0d * nodeStats.getPlanNodeCpuTime().toMillis() / totalCpuMillis;
+
+        output.append(format("CPU: %s (%s%%), Scheduled: %s (%s%%)",
```

Comment:
On L174 above and maybe in other places `cpu` is lowercase, here we use it in caps. It would be nice to be consistent.